build.boot

```diff
@@ -101,14 +101,19 @@
 ;; these edn files can hold additional information about the page such
 ;; as it's permalink identifier (`:page` key) or the page's title etc.
 
-(deftask test! []
+(deftask test!
+  "Run tests."
+  []
   (set-env! :source-paths #(conj % "test")
-            :dependencies #(into % '[[cljs-react-test "0.1.4-SNAPSHOT" :scope "test" :exclusions [cljsjs/react-with-addons]]
-                                     [cljsjs/react-with-addons "0.14.7-0"]]))
+            :dependencies #(into % '[[cljs-react-test "0.1.4-SNAPSHOT" :scope "test" :exclusions [cljsjs/react-with-addons]]]))
   (test-cljs :js-env :phantom
              :exit? true
              :update-fs? true
-             :namespaces #{"test.open-company-web.*"}))
+             :namespaces #{"test.open-company-web.*"}
+             :cljs-opts {:optimizations :whitespace
+                         :foreign-libs [{:provides ["cljsjs.react"]
+                                         :file "https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-with-addons.js"
+                                         :file-min "https://cdnjs.cloudflare.com/ajax/libs/react/15.4.2/react-with-addons.min.js"}]}))
```

Comment:
> Also :foreign-libs seems to be ignored w/ optimizations :none, it works only w/ :whitespace :simple and :advanced. Do you know anything about this?

That shouldn't be the case. I think many people use CLJSJS dependencies with `:none` and almost all CLJSJS libs use `:foreign-libs`. 

Regarding that issue mentioned—shouldn't you use the non-minified version as `:file-min` to fix that problem? If you really have the issue you linked then this is the right solution I guess :)